packages/dapp-svelte-wallet/react-ui/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <script src="lockdown.umd.js"></script>
+    <script>
+      if ('%NODE_ENV%' === 'production') {
```

Comment:
If we allow this to run in dev mode, we get the following error:
```
Uncaught TypeError: Cannot add property reactStack, object is not extensible
  at p (index.js:1)
  at K (index.js:1)
  at Module.de (index.js:1)
  at Object../node_modules/react-dev-utils/webpackHotDevClient.js (webpackHotDevClient.js:45)
  at __webpack_require__ (bootstrap:851)
  at fn (bootstrap:150)
  at Object.1 (logo.svg:1)
  at __webpack_require__ (bootstrap:851)
  at checkDeferredModules (bootstrap:45)
  at Array.webpackJsonpCallback [as push] (bootstrap:32)
  at main.chunk.js:1
```
![image](https://user-images.githubusercontent.com/8848650/134577756-91a94856-03ad-40d6-a15f-9c33be1d0acd.png)

Reply:
Hi @samsiegart could you either (or both) include the error text (not screenshot) when run against an uncompressed react? Or explain here how to reproduce.

@kriskowal @mhofman @dckc doesn't look like an override mistake. Looks like it is genuinely trying to monkey patch something.

Reply:
The node module itself comes already compressed, I might have some trouble trying to get an uncompressed version. To repro, you can remove the surrounding `if ('%NODE_ENV%' === 'production')` so that it runs in dev mode.

Then  `cd agoric-sdk/packages/dapp-svelte-wallet`, `agoric install`, `cd react-ui`, `yarn install` and `yarn start` to run local dev mode.

Reply:
to be clear, this error happens in production mode, not just dev? I'm only half surprised react attempts to monkey patch error (I remember reading something about them trying to capture the error stack)

Reply:
> this error happens in production mode / once built outside of a dev server?

The opposite.  The error only happens in non-prod mode if the code that's currently guarded by the if statement runs without that guard.

Reply:
This appears to be the culprit from the package source code (wasn't able to figure out how to import and run it uncompressed though): https://github.com/facebook/create-react-app/blob/bb64e31a81eb12d688c14713dce812143688750a/packages/react-error-overlay/src/effects/proxyConsole.js